CHANGELOG.org

```diff
@@ -2,6 +2,27 @@
 
 * Unreleased
 
+* 2022-05-25
+** Changed
+*** Update library versions using `clojure -T:search/outdated > outdated_$(date -u +%Y-%m-%d).org` command
+
+  | :name                                |   :current |    :latest |
+  |--------------------------------------+------------+------------|
+  | clj-kondo/clj-kondo                  | 2022.04.08 | 2022.04.25 |
```

Comment:
The updated version of clj-kondo in the GitHub action workflow does not seem to be included in this PR.  

TODO: Investigate if antq is still detecting for new versions in GitHub actions.

Reply:
antq is showing there is a new version of clj-kondo in the GitHub workflow, so this is updated along with other libraries to keep things consistent.